diff --git a/src/libsexp/include/sexp/sexp-error.h b/src/libsexp/include/sexp/sexp-error.h
--- a/src/libsexp/include/sexp/sexp-error.h
+++ b/src/libsexp/include/sexp/sexp-error.h
@@ -24,16 +24,17 @@
  * WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
  * OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
  * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
  */
 
 #pragma once
 
+#include <cstdint>
 #include <exception>
 #include <iostream>
 #include <string>
 
 namespace sexp {
 
 class sexp_exception_t : public std::exception {
   public:
diff --git a/version.txt b/version.txt
--- a/version.txt
+++ b/version.txt
@@ -1,1 +1,1 @@
-0.17.0
+0.17.0+PR2073
